src/main/java/success/planfit/domain/user/PlanfitUser.java

```diff
@@ -0,0 +1,34 @@
+package success.planfit.domain.user;
+
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+@Entity
+public class PlanfitUser {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(nullable = false)
+    private int user_id;
+
+    @Column(nullable = false)
+    private int login_id;
+
+    @Column(nullable = false)
+    private int password;
```

Comment:
```suggestion
    @Column(nullable = false, unique = true)
    private String loginId;

    @Column(nullable = false)
    private String password;
```

ERD를 확인해 보면 `planfit_user` 테이블의 `login_id`, `password` 컬럼의 자료형은 `varchar`라고 명시되어 있네요.
그에 맞게 자료형을 변경해주실 수 있나요?
(아직 미완성이어서 `int`로 두신 것이라면 무시해주세요!)

또한, 비즈니스 로직 상 `login_id` 컬럼은 중복되지 않는 값이어야 할 것 같습니다.
같은 경우에 해당하는 필드들에 위 예시 코드와 같이 `@Column` 애노태이션에 `unique = true` 속성을 추가해 주시거나, `@Table` 애노태이션의 `uniqueContraints` 속성을 사용해 유니크 제약조건을 걸어주면 감사하겠습니다!